### PR TITLE
add a new button that fetches the data from the servlet

### PR DESCRIPTION
**Pull request by menynexi**
I finally understand and got my head around the  fetching process. first a request is made to fetch content by clinking the button. The request is then turned into a promise which turns the data into a text and finally the test is sent back to be displayed in the container

https://mangutierrez-step-20202.uc.r.appspot.com

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -26,4 +26,10 @@ function addRandomGreeting() {
...
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
-}
+} 
+
+ function getCommentUsingArrowFunctions() {
+  fetch('/data').then(response => response.text()).then((comment) => {
+    document.getElementById('comment-container').innerText = comment;
+  });
+}
```
Comment (DirtGrubDylan) [inline portfolio/src/main/webapp/script.js:35]: New line at end of file

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -129,4 +129,10 @@ table, tr, td  {
     flex: 100%;
     max-width: 100%;
   }
+}
+
+.label{
+    background-color: #FF9900;
+    color:#cccccc;
+    font-size: 16px;
 }
```
Comment (DirtGrubDylan) [inline portfolio/src/main/webapp/style.css:138]: New line

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -129,4 +129,10 @@ table, tr, td  {
     flex: 100%;
     max-width: 100%;
   }
+}
+
+.label{
```
Comment (lzoebaker) [inline portfolio/src/main/webapp/style.css:134]: space before {

Comment (menynexi) [reply portfolio/src/main/webapp/style.css:134]: nice catch

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (menynexi) [reply portfolio/src/main/webapp/style.css:138]: can you clarify what you meant by "new line"

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (menynexi) [reply portfolio/src/main/webapp/script.js:35]: I understand, Thank you

**Pull request merged**